Update pro-develop-deploy-workflow.md (#6475)
magento/devdocs: src/cloud/architecture/pro-develop-deploy-workflow.md

### Merge to Staging

> Staging is a pre-production environment that provides all services and settings as close to the Production environment as possible. You should always push your code changes in integration to your Staging environment for full testing that includes all services. The first time you use the Staging environment, you must configure services, such as [Fastly CDN]({{ site.baseurl }}/cloud/cdn/cloud-fastly.html), [Blackfire Profiler]({{ site.baseurl }}/cloud/project/project-integrate-blackfire.html), and [New Relic]({{ site.baseurl }}/cloud/project/new-relic.html). Also, we recommend configuring payment gateways, shipping, notifications, and other vital services with sandbox or testing credentials.

Reviewer: ```suggestion
Staging is a pre-production environment that provides all services and settings as close to the Production environment as possible. You should always push your code changes from the Integration environment to the Staging environment so that you can perform thorough testing with all services. The first time you use the Staging environment, you must configure services, such as [Fastly CDN]({{ site.baseurl }}/cloud/cdn/cloud-fastly.html), [Blackfire Profiler]({{ site.baseurl }}/cloud/project/project-integrate-blackfire.html), and [New Relic]({{ site.baseurl }}/cloud/project/new-relic.html). Also, we recommend configuring payment gateways, shipping, notifications, and other vital services with sandbox or testing credentials.
```
